pkgs/development/python-modules/orgformat/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+}:
+
+buildPythonPackage {
+  pname = "orgformat";
+  version = "0-unstable-2024-10-23";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "novoid";
+    repo = "orgformat";
+    rev = "5346cc1a5fd670981e9b1d0bbd215eb5c79040d4";
+    hash = "sha256-4MnA+OzmEGN3KzjsZVwBXASiYTg529cfghpuf4owYJ8=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
```

Comment:
maybe add a check with this file somehow? https://github.com/novoid/orgformat/blob/master/run_unit_tests.sh

Not blocking for me, though. Just an idea!

Reply:
I tried figuring it out, but failed to do so. When I clone the repo and test there, the tests will pass, but will fail when building the nix package. There is test doing something with a timezone environment variable. Maybe this messes things up?